plugin/hover.py

```diff
@@ -120,8 +121,16 @@ def run_async() -> None:
 
     def request_symbol_hover_async(self, listener: AbstractViewListener, point: int) -> None:
         hover_promises = []  # type: List[Promise[ResolvedHover]]
-        document_position = text_document_position_params(self.view, point)
         for session in listener.sessions_async('hoverProvider'):
+            document_position = text_document_position_params(self.view, point)
+            range_hover_provider = session.get_capability('experimental.rangeHoverProvider')
+            if range_hover_provider:
+                region = first_selection_region(self.view)
+                if region is not None:
+                    if region.contains(point):
+                        # hovering selection or only text was selected
+                        document_position.pop('position', None)
+                        document_position['range'] = region_to_range(self.view, region).to_lsp()
```

Comment:
OK mypy isn't happy about these two lines.
```
plugin/hover.py:132: error: Key 'position' of TypedDict "TextDocumentPositionParams" cannot be deleted
plugin/hover.py:133: error: TypedDict "TextDocumentPositionParams" has no key 'range'
```
From my I see this one is a known issue in mypy https://github.com/python/mypy/issues/8887
Should simply disable the checks or any other workaround ?